### ISSUE TITLE
close-explicitly prop

### ISSUE THREAD

**Issue by choper725** [inertiaui/modal#36]
close-explicitly prop seems not working.

**Comment by pascalbaljet** [comment]
Can you provide some example code of how you are using it?

**Comment by choper725** [comment]
sure !
just like in the docs
on calling modal-link
![image](https://github.com/user-attachments/assets/9b8b2e52-bbe3-418a-9dd0-baf7160d9dcf)


on rendered page
there is a a Modal in template with no props (tried putting it there also but still the same)

![image](https://github.com/user-attachments/assets/b0116ce7-d8b9-4b60-b0c7-d94462754237)

**Comment by Cannonb4ll** [comment]
I am indeed experiencing the exact same. Created a link exactly the same as the docs:

```html
<template>
    <ModalLink href="/users/create" :close-explicitly="true">
        Create User
    </ModalLink>
</template>
```

Doesn't seem to honor this setting. I also wonder if this should be a setting on the link, or maybe on the modal component itself.

**Comment by pascalbaljet** [comment]
Fixed in v0.6.2
